digdag-ui/model.js

```diff
@@ -297,6 +297,15 @@ export class Model {
     })
   }
 
+  retrySession (session: Session) {
+    const { lastAttempt } = session
+    return this.put('attempts', {
```

Comment:
I also think client-side should generate an unique id so that retrying submission is idempotent. Just setting a random UUID (UUID v4) to the field is good enough.